src/Microsoft.VisualStudio.Editors/PropPages/Strings.resx

```diff
@@ -131,4 +131,15 @@
     <value>http://go.microsoft.com/fwlink/?LinkID=287120</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="LanguageVersionAutomaticallySelected" xml:space="preserve">
+    <value>Automatically selected based on framework version</value>
+    <comment>Refers to the C# language version being automatically selected based on the target framework</comment>
+  </data>
+  <data name="CantSelectLanguageVersionLinkLabel" xml:space="preserve">
+    <value>Why can't I select a different C# version?</value>
+  </data>
+  <data name="CantSelectLanguageVersionFWLink" xml:space="preserve">
+    <value>https://aka.ms/csharp-versions</value>
```

Comment:
This document doesn't currently reflect this change. Is that work being tracked somewhere?

Reply:
That's work @cartermp is planning on doing.

Reply:
Yep.